demisto_sdk/commands/common/schemas/playbook.yml

```diff
@@ -129,8 +129,11 @@ mapping:
                   simple:
                     type: text
               tags:
+                type: map
                 required: false
```

Comment:
you might need `allowempty` here as well - you can check that by running validate on a playbook that has this situation:
![image](https://user-images.githubusercontent.com/50295826/94795464-04a1b500-03e6-11eb-88e8-9b0fc9610465.png)

Reply:
No need, the validation passed ok.
Ran a build using this branch
![image](https://user-images.githubusercontent.com/45915502/94800798-c4decb80-03ed-11eb-9d50-6107f89ca1e7.png)
Failed on other stuff of course, not relevant to this,

Reply:
it is not needed to fix the issue, but it is needed for the case a playbook with empty tags is given
lets add it please